src/sql/workbench/contrib/notebook/browser/notebookEditor.ts

```diff
@@ -424,11 +428,6 @@ export class NotebookEditor extends BaseEditor implements IFindNotebookControlle
 		}
 	}
 
-	private updatePosition(range: NotebookRange): void {
```

Comment:
This method was just a duplicate implementation of setSelection method so removed this private method and changed references to this method over to setSelection method.